Add TextCleaner plugin

valentineus opened the pull request:
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] If my package is a syntax it is named after the language it supports (without suffixes like "syntax" or "highlighting").

My package is a Sublime Text 4 plugin that cleans and standardizes formatting.

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

braver commented:
The bot says:
> Packages must be sorted alphabetically (by name)

Also:
- Your command is “clear text”, not “clean text… a typo?
- It’s recommended to prefix the command with the name of the providing package, so users know where it came from (and which settings to change to make it behave differently). E.g. “TextCleaner: clean text”.
- You should expose your settings via the package settings menu, and the command palette.

braver commented:
Please respond to the feedback to continue the review.

valentineus commented:
> Please respond to the feedback to continue the review.

Thank you for your feedback! I will do my best to send all the corrections addressing your comments within a week.

braver commented:
> within a week

Had a chance to handle the feedback already?

braver commented:
We can always re-open the PR when you're ready.
